### DISCUSSION TITLE
[Unstable] rtAfflictionTypes

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15237] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

HumanDefaultRagdoll.xml
line 34

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Nice catch! Fixed in our private repo now, fix coming up in the next unstable build.
